07LessonSeven/homework/css/style.css

```diff
@@ -1,59 +1,82 @@
 body {
-  font-family: sans-serif;
-  font-weight: lighter;
+  font-family: sans-serif, Arial;
+  font-size: 14px;
 }
+
+#container {
+  display: block;
```

Comment:
You don't need to set `display: block` as the semantic element are already block elements, you did this quite a bit in your code. You don't want to write unnecessary code that you will have to maintain, the less code the better. Great job on the rest of the CSS.